org.lflang/src/org/lflang/util/StringUtil.java

```diff
@@ -79,70 +79,72 @@ public static String removeQuotes(String str) {
         }
         return str;
     }
-    
```

Comment:
This class appears to have an overlap in purpose with the new `org.lflang/src/org/lflang/ast/FormattingUtils.java`. Would it make sense to merge them or scope them more clearly?